[2021-01-21 21:57:22,445] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_1 2021-01-21T21:57:15.033646+00:00 [queued]>
[2021-01-21 21:57:22,451] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_1 2021-01-21T21:57:15.033646+00:00 [queued]>
[2021-01-21 21:57:22,451] {taskinstance.py:1017} INFO - 
--------------------------------------------------------------------------------
[2021-01-21 21:57:22,451] {taskinstance.py:1018} INFO - Starting attempt 1 of 1
[2021-01-21 21:57:22,451] {taskinstance.py:1019} INFO - 
--------------------------------------------------------------------------------
[2021-01-21 21:57:22,460] {taskinstance.py:1038} INFO - Executing <Task(BashOperator): task_1> on 2021-01-21T21:57:15.033646+00:00
[2021-01-21 21:57:22,461] {standard_task_runner.py:51} INFO - Started process 33487 to run task
[2021-01-21 21:57:22,465] {standard_task_runner.py:75} INFO - Running: ['airflow', 'tasks', 'run', 'parallel_dag', 'task_1', '2021-01-21T21:57:15.033646+00:00', '--job-id', '3', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/parallel_dags.py', '--cfg-path', '/tmp/tmp5og17uot']
[2021-01-21 21:57:22,465] {standard_task_runner.py:76} INFO - Job 3: Subtask task_1
[2021-01-21 21:57:22,492] {logging_mixin.py:103} INFO - Running <TaskInstance: parallel_dag.task_1 2021-01-21T21:57:15.033646+00:00 [running]> on host ip-172-31-18-6.ap-south-1.compute.internal
[2021-01-21 21:57:22,514] {taskinstance.py:1232} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=parallel_dag
AIRFLOW_CTX_TASK_ID=task_1
AIRFLOW_CTX_EXECUTION_DATE=2021-01-21T21:57:15.033646+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-01-21T21:57:15.033646+00:00
[2021-01-21 21:57:22,514] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2021-01-21 21:57:22,514] {bash.py:158} INFO - Running command: sleep 3
[2021-01-21 21:57:22,518] {bash.py:169} INFO - Output:
[2021-01-21 21:57:25,520] {bash.py:177} INFO - Command exited with return code 0
[2021-01-21 21:57:25,536] {taskinstance.py:1142} INFO - Marking task as SUCCESS. dag_id=parallel_dag, task_id=task_1, execution_date=20210121T215715, start_date=20210121T215722, end_date=20210121T215725
[2021-01-21 21:57:25,558] {taskinstance.py:1195} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-01-21 21:57:25,568] {local_task_job.py:118} INFO - Task exited with return code 0
